openedx/core/djangoapps/content_tagging/api.py

```diff
@@ -174,41 +188,94 @@ def get_all_object_tags(
     return grouped_object_tags, taxonomies
 
 
-def set_object_tags(
+def set_all_object_tags(
     content_key: ContentKey,
-    object_tags: TagValuesByTaxonomyExportIdDict,
+    object_tags: TagValuesByTaxonomyIdDict,
```

Comment:
I was thinking we may still need the version of this that uses `export_id` for the course .tar.gz import/export process, though I don't see it on https://github.com/openedx/edx-platform/pull/34356/files so I guess it's fine to change this.

Reply:
Ya, I was surprised that nothing was using this method -- but pleased that I could repurpose it :)